@here/harp-lines/lib/Lines.ts

```diff
@@ -151,6 +152,7 @@ export function createLineGeometry(
         sum = sum + len;
         segments[i + 1] = sum;
     }
+    const lineLength = segments[segments.length - 1];
```

Comment:
If this is constant, couldn't just that be a uniform passed down to the material?

Reply:
We can't use uniform because this method is used in LineGroup class that renders a bunch of lines at once, those lines will have different length.